draft-ietf-quic-multipath.md

```diff
@@ -1194,15 +1194,15 @@ per path ID. However, depending on which path acknowledgements are
 sent, the actual RTT of a path cannot be calculated or may not be
 the right value to be used.
 
-Instead of using RTT, we define the acknowledgment delays as
+Instead of using the real RTT of a path, it is recommended to use
 the sum of two one-way delays: the delay
```

Comment:
instead of "use", I would say "consider". We don't want implementations to develop estimators of one-way delays!